api/kessel/relations/v1beta1/common.proto

```diff
@@ -46,6 +46,6 @@ message ObjectReference {
 }
 
 message ObjectType {
-	string namespace = 1 [(validate.rules).string.min_len = 1];
+	optional string namespace = 1 [(validate.rules).string.min_len = 1];
```

Comment:
```suggestion
	optional string namespace = 1;
```
Does the validation rule cause any issues being present? I'm not sure if `optional` alone will fix the issue

Reply:
Thanks! I just updated it.

Yea, it is throwing validation errors from our python client because we only have name specified, e.g., https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/migration_tool/utils.py#L42